src/App.tsx

```diff
@@ -9,31 +10,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(0);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is&nbsp;Tab 1
+        {`Selected tab is ${tabs[selectedTab].title}`}
       </h1>
 
       <div>
         <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active">
-              <a href="#tab-1">Tab 1</a>
-            </li>
-
-            <li className="">
-              <a href="#tab-2">Tab 2</a>
-            </li>
-
-            <li className="">
-              <a href="#tab-3">Tab 3</a>
-            </li>
-          </ul>
+          <Tabs
+            tabs={tabs}
+            select={selectedTab}
+            onChange={(index:number) => {
+              setSelectedTab(index);
+            }}
```

Comment:
You haven't used the names from the checklist, but it's ok I think:
![image](https://user-images.githubusercontent.com/33294729/184901976-c3a6279d-74a3-44c3-9a48-a68b35e049e7.png)

Reply:
++